README.md

```diff
@@ -30,13 +30,19 @@ A container engine is needed locally such as:
 
 **Note**: Set the `DOCKER_HOST` env var property using `podman` to let idpbuilder to talk with the engine (e.g  export DOCKER_HOST="unix:///var/run/docker.sock")
 
-## Quickstart
+## Getting Started
 
 If you are interested in running idpbuilder in Codespaces through your browser, check out the [Codespaces](#running-in-codespaces) section.
 
-### Download and install the idpbuilder
+### Quick Install
 
-Download the latest release with the commands:
+You can *carefully* run the following script to install `idpbuilder`
```

Comment:
I am not sure why adding `*carefully*` here is adding. How are we advocating that folks carefully run the script?

Reply:
I think the intent is to make people aware that running this type of command *could* be dangerous if they don't review the script. Maybe: 

We offer this command as a convenience for installing idpbuilder (be sure to check the script first if you are concerned):

Reply:
good call. will update the info there.